GraceGardner/rock-paper-scissors
# Feature/winner display

#### What's this PR do?  
This PR adds functionality that shows who won a game. 

#### Where should the reviewer start?
With opening the index.html from terminal

#### How should this be manually tested?
Play a few rounds and check that the emoji is showing up on only the selected character. Then make sure that the player won or lost is properly displayed after the round. 


#### Any background context you want to provide?
When developing this functionality I struggled with keeping the same button from being selected multiple times in a row. 

#### Screenshots (if appropriate)
#### Questions:
